src/tests/ftest/daos_test/suite.yaml

```diff
@@ -42,6 +42,7 @@ pool:
 server_config:
   engines_per_host: 2
   name: daos_server
+  crt_timeout: 10
```

Comment:
Since https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/server_utils_params.py#L400-L401 still exists you'll end up with both a `crt_timeout: 10` server-level entry and a `crt_timeout: 30` entry per engine.  It would appear that the per-engine entry takes precedent.  Have you attempted this with L400-L401 removed?

Reply:
Thanks. Trying this in #8434.